infra/serverless/lambda-web-app-test/README.md

```diff
@@ -0,0 +1,26 @@
+# Serverless App Example - Weather Forecast API
+
+This folder contains the infrastructure and application code for a simple application that provides weather forecast details using API Gateway, Lambda and S3.
+
+## Functional Details
+
+This is an application that allows users to retrieve weather forecast data via their browser. It consists of a front-end UI (client-side), and a backend using a function exposed via an API (server-side). Parameters entered by the user are passed from the front-end, to the function in the backend, which then calls an API to retrieve the weather forecast data based on the user's inputs.
+
+The client side part of the application should sit on S3. The server side of the application sits on Lambda, exposed via API Gateway. The client-side part of the application uses the API Gateway endpoint to interact with the server-side part of the application.
+
+Seeing as the front end (client side) contains JavaScript that manipulates data from the backend (server side) with another origin, this example application must support Cross-Origin Resource Sharing (CORS). Refer to the terraform script `main.tf` to see how the API Gateway infrastructure is set up to support CORS.
+
+## What this folder includes
+
+### Infrastructure
+
+- A Terraform file for provisioning API Gateway and Lambda resources, including required IAM permissions.
+
+### Application
+
+- A Zip file that includes the Node.js files required for the application run on the Lambda function
```

Comment:
Can you add the code for the zip file to this PR. If you look at the `build` and `full-release` script commands in package.json you can see how you can transform a set of typescript files into a single index.js and zip the generated .js files into a local zip.

This would allow for local development to not rely on an uploaded zip file which requires manual updating when code changes are made (external to this repository).

See: [esbuild](https://esbuild.github.io/getting-started/) & [this line of code](https://github.com/nia-vf/cloudboyz/blob/main/package.json#L13)